Mathlib/SetTheory/Nimber/Field.lean

```diff
@@ -8,15 +8,14 @@ import Mathlib.SetTheory.Nimber.Basic
 import Mathlib.Tactic.Abel
 
 /-!
-# Nimber multiplication
+# Nimber multiplication and division
 
 The nim product `a * b` is recursively defined as the least nimber not equal to any
 `a' * b + a * b' + a' * b'` for `a' < a` and `b' < b`. When endowed with this operation, the nimbers
 form a field.
 
 ## Todo
 
-- Define nim division and prove nimbers are a field.
 - Show the nimbers are algebraically closed.
```

Comment:
The definition you've given for inv is very nice; I think an implementation note to explain what you did (essentially mimicking the docstring for invAux and mentioning that it's mutually recursive) could be nice.

Reply:
This definition is "standard", in the sense that every reference which defines the nimber inverse explicitly uses it. Though a few references instead define it indirectly via the ironically more technical simplicity theorem. I'll add a docstring explaining this.

Reply:
Oh sorry, I mean your implementation of the definition (mutual induction and then putting an if/then on top)